docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -712,10 +718,6 @@ public void pullImage(String imageName) {
 
             PullImageCmd pullImageCmd = this.dockerClient.pullImageCmd(image.getName());
 
-            if (this.cubeConfiguration.getDockerRegistry() != null) {
-                pullImageCmd.withRegistry(this.cubeConfiguration.getDockerRegistry());
```

Comment:
No, before we were just attaching the repo url to the image name. It is a way to append the url of registry to image name programmatically. But this should be already provided by user on docker-compose.